### DISCUSSION TITLE
Formatting NumberInput as currency on blur 🤔

### DISCUSSION THREAD

**Discussion by Sajarin-M** [mantinedev/mantine#3322] [Q&A]
I was wondering if it is possible to use the `formatter` prop in a `NumberInput` component to display the input as a currency with a thousands separator (e.g. `10,000`), but only apply the formatting when the input is blurred (not while the user is typing). If this is not possible with the `formatter` prop, I would be grateful for any recommendations on how to achieve this behavior. Thank you in advance for your time and assistance. 😊

**Comment by thanseer-pk** [comment]
Hi @Sajarin-M I am also having this problem. Any luck fixing it?
